test/runner.py

```diff
@@ -2,8 +2,6 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
this helps a lot in not accidentaly testing the installed version instead of the version in your current directory.